lib/topological_inventory/orchestrator/source_type.rb

```diff
@@ -27,11 +26,11 @@ def collector_definition(collector_image_tag = 'latest')
       end
 
       def supported_source_type?
-        attributes['name'].present? && SUPPORTED_TYPES.include?(attributes['name'])
+        attributes['name'].present? && self[:enabled?]
       end
 
       # Methods amazon? / ansible_tower? / azure? / openshift?
-      SUPPORTED_TYPES.each do |type|
+      %w[amazon ansible-tower azure openshift].each do |type|
```

Comment:
Does it still make sense to keep the list of supported source type in a constant? I can see us continuing to add to that list as we board additional source types that require collectors and it would be easier to add them to the constant.

Reply:
yeah, there we can generate just one method returning true (based on `attributes['name']`). 
Other methods with `?` can be handled by `method_missing`. Better in `is_..._?` format, if it won't take much time.